VoodooI2CHID/VoodooI2CMultitouchHIDEventDriver.cpp

```diff
@@ -597,7 +597,7 @@ IOReturn VoodooI2CMultitouchHIDEventDriver::parseDigitizerElement(IOHIDElement*
     return kIOReturnSuccess;
 }
     
-IOReturn VoodooI2CMultitouchHIDEventDriver::parseElements() {
+IOReturn VoodooI2CMultitouchHIDEventDriver::parseElements(UInt32 forUsage) {
```

Comment:
Can you please consider a name change to just usage (see https://developer.apple.com/documentation/kernel/iohidelement/1426848-conformsto)